calendar/README.md

```diff
@@ -0,0 +1,171 @@
+Ballerina Connector For Microsoft Calendar
```

Comment:
README.md inside calendar directory is not required .

Reply:
Removed the file